lib/control/v1/conqueso.js

```diff
@@ -48,6 +49,7 @@ function injectConquesoAddresses(properties) {
 
       conquesoIps[`conqueso.${cluster}.ips`] = properties.consul[service].addresses.join(',');
     });
+    delete properties.consul; // eslint-disable-line no-param-reassign
```

Comment:
It looks like `properties` a reference to the object managed by the Storage module. Mutating it here is likely to have strange effects on other consumers that expect it's `consul` key to not disappear whenever this handler is executed.

We should do something like this insterad:

```
// Construct a new Object with only 'conqueso.SERVICE.ips=list,of,addresses` key/values
const conquesoIps = translateConquesoAddresses(properties);

const conquesoProperties = Object.assign({}, properties, conquesoIps);
delete conquesoProperties.consul;

makeJavaProperties(flatten(conquesoProperties));
...
```